stuhlandia.py

```diff
@@ -0,0 +1,21 @@
+row1 = set('qwertyuiop\n')
+row2 = set('asdfghjkl\n')
+row3 = set('zxcvbnm\n')
+
+try:
+    with open("states.txt", "r") as f:
```

Comment:
include all of this in a function

Reply:
in that function, separate the code that uses opens the file from the function logic